### PR TITLE
Work around #1769 by doing the DPI/height calculation ourselves.

### PR DESCRIPTION
**Pull request by DHowett-MSFT**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1770)

### CODE REVIEW

**Comment by msft-Jeyaram** [comment]
:shipit:

**Comment by MSFTFox** [comment]
:shipit:

**Comment by msft-Jeyaram** [comment]
Could we get this in today and spin off a full build 👯‍♂️

File: Frameworks/CoreGraphics/CGContext.mm [hunk 1]
```diff
@@ -284,8 +284,19 @@ inline HRESULT _RestoreD2DDrawingState(ID2D1DrawingStateBlock* drawingState) {
...
         // CG is a lower-left origin system (LLO), but D2D is upper left (ULO).
         // We have to translate the render area back onscreen and flip it up to ULO.
+        float height = 0.f;
+#ifdef _M_ARM
+        // TODO: GH#1769; GetSize() returns garbage on ARM.
+        // We would prefer to allow D2D to do the DPI calculation.
+        D2D1_SIZE_U targetPixelSize = deviceContext->GetPixelSize();
+        FLOAT dpiX, dpiY;
+        deviceContext->GetDpi(&dpiX, &dpiY);
+        height = (targetPixelSize.height * 96.0) / dpiY;
```
Comment (rajsesh-msft) [inline Frameworks/CoreGraphics/CGContext.mm]: Nit: magic number.

Comment (DHowett-MSFT) [inline Frameworks/CoreGraphics/CGContext.mm]: I agree, but this is fairly localized :)

**Pull request merged**